Source/AliveLibAO/Factory.cpp

```diff
@@ -460,9 +461,25 @@ EXPORT void Factory_ChimeLock_4870D0(Path_TLV* /*pTlv*/, Map* /*pMap*/, TlvItemI
 }
 
 
-EXPORT void Factory_ElumStart_Unknown_4873D0(Path_TLV* /*pTlv*/, Map* /*pMap*/, TlvItemInfoUnion /*tlvOffsetLevelIdPathId*/, __int16 /*loadMode*/)
+ResourcesToLoadList unk_4BD418[4] = { //todo
+    { 3, 0x6D696E41 },
+    { 200, 0x6D696E41 },
+    { 230, 0x6D696E41 },
+    { 216, 0 }
```

Comment:
shouldn't there be a null entry after this? isn't that how it knows what the size of the array is?